man/sd_bus_call.xml

```diff
@@ -104,7 +104,12 @@
     <refsect2 id='errors'>
       <title>Errors</title>
 
-      <para>Returned errors may indicate the following problems:</para>
+      <para>Returned errors may indicate the following problems. Note that when
+      <function>sd_bus_call()</function> fails and because it received a D-Bus error reply, the function
+      may return the error code mapped from the reply. In that case, <parameter>ret_error</parameter>
+      may contain a description about the error. See
+      <citerefentry><refentrytitle>sd_bus_error</refentrytitle><manvolnum>3</manvolnum></citerefentry>.
```

Comment:
This is unclear.
"fails and because" → "fails because" ?

All the "may return" hedges make it hard to figure out what is going on. I would expect a concrete declaration:
`When <function>sd_bus_call()</function internally receives a D-Bus error reply, it will set <parameter>ret_error</parameter> if ..., and will return a negative value when ... .`

Reply:
> All the "may return" hedges make it hard to figure out what is going on.

Well it was kind of hard to figure out what sd_bus_call returns. Basically it can return same set of errors as sd_bus_call_async for errors happened inside the function. However, unlike the async function as a last step it will map any dbus error message it received to an error code and return that code.